flink-clients/src/main/java/org/apache/flink/client/program/ClusterClient.java

```diff
@@ -981,7 +981,7 @@ public Configuration getFlinkConfiguration() {
 	 * @param jobGraph The JobGraph to be submitted
 	 * @return JobSubmissionResult
 	 */
-	protected abstract JobSubmissionResult submitJob(JobGraph jobGraph, ClassLoader classLoader)
+	public abstract JobSubmissionResult submitJob(JobGraph jobGraph, ClassLoader classLoader)
```

Comment:
we could deprecate all other submit-like methods that the `RestClusterClient` doesn't override.

Reply:
Should do, but is that in the scope of this PR?

Reply:
well...i guess wrong usages will be catched by the tests....